packages/api/prompts/cell-generator-javascript.txt

```diff
@@ -0,0 +1,128 @@
+===== BEGIN INSTRUCTIONS CONTEXT =====
+
+You are tasked with generating a Srcbook code cell for the user.
+
+A Srcbook is a JavaScript notebook following a markdown-compatible format called .srcmd. It's an interactive and rich way of programming that follows the literate programming idea. 
```

Comment:
>  It's an interactive and rich way of programming that follows the literate programming idea.

These prompts feel too wordy IMO. This ^, for example, doesn't seem necessary.

Reply:
Yes, the prompts can definitely be iterated on. This made more sense in the generate srcbook one...I'll remove that particular sentence

Without evals, it's a bit of a nacked process. I am happy to remove this, but the next step for prompt iteration is to set up some basic evals. For this particular flow, the evals could programmatically test that we get valid srcbook code back based on a set of known user request examples. 


Once we have that, we can start iterating with a bit more confidence.